src/main/java/hudson/plugins/performance/PerformancePublisher.java

```diff
@@ -581,6 +582,61 @@ private Result checkAverageResponseTime(PerformanceReport performanceReport, Has
         return null;
     }
 
+    // write 'standard mode' report in xml containing each of the populated columns otherwise found in the jelly output.
+    // producing an on disk, machine parseable format allows for archiving and automation.
+    private void writeStandardResultsToXML(Run<?, ?> run, Collection<PerformanceReport> parsedReports) throws IOException {
+        FileWriter fw = null;
+        BufferedWriter bw = null;
+        try {
+            xmlDir = run.getRootDir().getAbsolutePath();
+            xmlDir += "/" + archive_directory;
+
+            if (!new File(xmlDir).exists()) {
+                new File(xmlDir).mkdirs();
+            }
+
+            xmlfile = new File(xmlDir + "/standardResults.xml");
+            xmlfile.createNewFile();
```

Comment:
You can use `org.apache.commons.io.FileUtils#touch` for create file and parent directories